src/types.ts

```
@@ -271,6 +271,24 @@ export interface WatchEventListenerOptions extends AddEventListenerOptions {
+ */
+export type EventHandlerResult =
+  | void
+  | Promise<void>
+  | Generator<any, void, any>
+  | AsyncGenerator<any, void, any>;
+
+/**
+ * A generic event handler function type.
+ * @template E - The specific Event type (e.g., MouseEvent, KeyboardEvent).
+ */
+export type EventHandler<E extends Event = Event> = (event: E) => EventHandlerResult;
```

Biome (2.1.2)

[error] Shouldn't redeclare 'EventHandler'. Consider to delete it or rename it.

'EventHandler' is defined here:


(lint/suspicious/noRedeclare)